src/seedu/addressbook/AddressBook.java

```diff
@@ -217,6 +217,10 @@ public static void main(String[] args) {
 		showWelcomeMessage();
 		processProgramArgs(args);
 		loadDataFromStorage();
+		performAllFunctions();
+	}
+
+	private static void performAllFunctions() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.